example/src/scenarios/cross-device-flow-with-rp.tsx

```diff
@@ -71,10 +75,26 @@ export default async () => {
     // Validate Request object signature
     // TODO
 
+    // select claims to be disclose from pid
+    // these would be selected by users in the UI
+    const claims = [
+      "unique_id",
+      "given_name",
+      "family_name",
+      "birthdate",
+      "place_of_birth",
+      "tax_id_number",
+      "evidence",
+    ];
+
+    // verified presentation is signed using the same key of the wallet attestation
+    const unsignedVpToken = RP.prepareVpToken(requestObj, [pidToken, claims]);
+    const vpToken = await sign(unsignedVpToken, walletInstanceKeyTag);
```

Comment:
This is not `vpToken` but only signature

Reply:
Done in https://github.com/pagopa/io-react-native-wallet/pull/20/commits/598a297

Reply:
In the PR I don't see the fix commit.